.gitignore

```diff
@@ -1,9 +1,12 @@
+/.*
+!/.gitignore
+!/.eslintrc.json
+!/.github
```

Comment:
Do we need these or any changes to this file?

Reply:
The original `.gitignore` was

```
/bower_components/
/node_modules/
/.pulp-cache/
/output/
/generated-docs/
/.psc*
/.purs*
/.psa*
package-lock.json
```

However the things like `/.pulp-cache/` are no longer relevant after moving to the new `spago` so I thought the `.gitignore` could use a refresh